src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -416,7 +423,11 @@ public interface IStartConfiguration
             TaskProcessMode ProcessMode { get; set; }
         }
 
-      
+        public interface IStartConfiguration2 : IStartConfiguration
```

Comment:
https://github.com/DynamoDS/Dynamo/wiki/API-Changes

seems I forgot to add some of the other changes we did as well.